lib/dependency-manager-adapters/npm.js

```diff
@@ -77,7 +77,7 @@ module.exports = CoreObject.extend({
     if (!this.useYarnCommand) {
       try {
         if (fs.statSync(path.join(this.cwd, this.yarnLock)).isFile()) {
-          ui.writeLine(chalk.yellow('Detected a yarn.lock file, add useYarn: true to your configuration if you want to use Yarn to install npm dependencies.'));
+          ui.writeLine(chalk.yellow('Detected a yarn.lock file. Add `useYarn: true` to your `ember-try.js` configuration file if you want to use Yarn to install npm dependencies.'));
```

Comment:
Can you update to say `config/ember-try.js` (new folks may not know where the file is either)?

Reply:
Good point, done :)